docs/source/index.rst

```diff
@@ -273,7 +273,7 @@ Deployment Guides and Tutorials
     deep-learning/index
     traditional-ml/index
     deployment/index
-    tracking
+    tracking/index
```

Comment:
We'll want to create a stub redirect page similar to what was done for `registry` / `model-registry` in order to not break external links to `tracking#<x>` (I'm sure there are links in Databricks docs that point to stuff in this page). Can you take a look at how the registry redirect was done and ensure that we're doing a 'hidden' page (not listed in ToC) that preserves the old name that will redirect to the content of the refactored main page (index.html)?

Reply:
Totally makess sense, will follow the same way as registry page

Reply:
@BenWilson2 It seems the approach of [registry](https://github.com/mlflow/mlflow/blob/master/docs/source/registry.rst) page doesn't work here as the old/new docs are in different directory. Like if we do this:

```
:orphan:
.. include:: tracking/index.rst
  :start-line: 1
```

it simply copies the scripts in `tracking/index.rst` to `tracking.rst`, then all relative path references will break e.g. static resources.

I also tried [sphinxcontrib-redirects](https://pypi.org/project/sphinxcontrib-redirects/) plugin, but unfortunately it only supports page level redirection and doesn't keep section `tracking#<x>`.

We might be able to implement custom JS solution, but since we will be migrating to markdown-based framework, I'm afraid it will be a thrown-away anyway. Hence, in this PR I will just keep `tracking.rst` file instead of `tracking/index.rst`, and just include child pages in `tracking/` directory.